graveler/committed/tree/parts_and_values.go

```diff
@@ -0,0 +1,78 @@
+package tree
+
+import "github.com/treeverse/lakefs/graveler"
+
+type partsAndValuesIterator struct {
```

Comment:
Suggestion shorter name - TreeValueIterator

Reply:
I suggest: TreeIterator for this one,
and the other iterator (which only returns values) will be TreeValueIterator

Reply:
And since we're under the tree -pun not intended- package, how about just Iterator.